### PR TITLE
Create uri-from-top-55-most-popular-apps.txt

### PR DESCRIPTION
**Pull request by J-GainSec**
Removed trailing slashes

### CODE REVIEW

**Review by ItsIgnacioPortal — CHANGES_REQUESTED** [review]
I believe you can use the following script to remove the trailing slashes. I haven't tested it myself though
 https://github.com/danielmiessler/SecLists/commit/791bf1bf1d11e2415e2410c8aeda5a2ba0da359a

File: Discovery/Web-Content/uri-from-top-55-most-popular-apps.txt [hunk 1]
```diff
@@ -0,0 +1,2418 @@
...
+/..
+/.DS_Store
+/.codepushrelease
+/0-.
+/10000000
+/3-
+/311:
+/3126
+/319:
+/345
```
[the remark this answers is not in the document]
Comment (J-GainSec) [reply Discovery/Web-Content/uri-from-top-55-most-popular-apps.txt:14]: Trailing slashes are at the end of the. Leading slashes are in the front. I think you mean remove the leading slashes. Removing them now

File: Discovery/Web-Content/uri-from-top-55-most-popular-apps.txt [hunk 2]
```diff
@@ -0,0 +1,2418 @@
...
+RFC3211Wrap
+RSA
+SIC
+SSL3MAC
+Short
+ShortArray
+String
+Superuser
+Throwable
+U^JVTP\rdTbd
```
Comment (ItsIgnacioPortal) [inline Discovery/Web-Content/uri-from-top-55-most-popular-apps.txt:77]: This looks like noise

File: Discovery/Web-Content/uri-from-top-55-most-popular-apps.txt [hunk 3]
```diff
@@ -0,0 +1,2418 @@
...
+data/local/bin/su
+data/local/su
+data/local/tmp/adjust.preinstall
+data/local/tmp/exopackage
+data/local/tmp/exopackage/%s//secondary-dex
+data/local/tmp/pre_install.appsflyer
+data/local/xbin
+data/local/xbin/su
+datafiles/%s.json
+ddYWah^Y8E=
```
Comment (ItsIgnacioPortal) [inline Discovery/Web-Content/uri-from-top-55-most-popular-apps.txt:286]: This looks like noise

File: Discovery/Web-Content/uri-from-top-55-most-popular-apps.txt [hunk 4]
```diff
@@ -0,0 +1,2418 @@
...
+audio/x-pn-realaudio
+audio/x-realaudio
+audio/x-scpls
+audio/x-sd2
+audio/x-unknown
+audio/x-wav
+auth/logout
+authenticate/errors
+autocomplete/json
+bb\u0015\\fjf
```
Comment (ItsIgnacioPortal) [inline Discovery/Web-Content/uri-from-top-55-most-popular-apps.txt:980]: This looks like noise

File: Discovery/Web-Content/uri-from-top-55-most-popular-apps.txt [hunk 5]
```diff
@@ -0,0 +1,2418 @@
...
+friendships/destroy.json
+friendships/destroy.json?user_id=
+friendships/incoming.json?cursor=
+friendships/lookup.json?screen_name=
+friendships/lookup.json?user_id=
+friendships/no_retweets/ids.json
+friendships/outgoing.json?cursor=
+friendships/show.json
+friendships/update.json
+g_m\u001cfq\u001fnvno
```
Comment (ItsIgnacioPortal) [inline Discovery/Web-Content/uri-from-top-55-most-popular-apps.txt:1126]: This looks like noise

**Review by ItsIgnacioPortal — CHANGES_REQUESTED** [review]
~~Please remove these seemingly random strings~~

On second look, a couple of them look like UNICODE encoding. Could you verify?

File: Discovery/Web-Content/uri-from-top-55-most-popular-apps.txt (hunk 5 above)
Comment (J-GainSec) [reply Discovery/Web-Content/uri-from-top-55-most-popular-apps.txt:1126]: I don't think line 1126 is noise but probably unique to that application. Will remove

**Review by J-GainSec — COMMENTED** [review]
Removed a few useless entries as requested

**Review by ItsIgnacioPortal — APPROVED** [review]
Thanks for contributing :)

**Comment by g0tmi1k** [comment]
How was this list made?
Whats the source?

**Comment by J-GainSec** [comment]
https://github.com/danielmiessler/SecLists/pull/781#issuecomment-1168353194

https://twitter.com/nullenc0de/status/1425973675715612672

https://gist.github.com/nullenc0de/e9d1f2a8a0a38c9bfcb5bdb9fc7191ea

**Comment by g0tmi1k** [comment]
Thank you!

**Pull request merged**